app/code/Magento/Sales/Model/Order/CreditmemoFactory.php

```diff
@@ -42,17 +49,20 @@ class CreditmemoFactory
      * @param \Magento\Sales\Model\Convert\OrderFactory $convertOrderFactory
      * @param \Magento\Tax\Model\Config $taxConfig
      * @param \Magento\Framework\Serialize\Serializer\Json $serializer
+     * @param FormatInterface $localeFormat
      */
     public function __construct(
         \Magento\Sales\Model\Convert\OrderFactory $convertOrderFactory,
         \Magento\Tax\Model\Config $taxConfig,
-        \Magento\Framework\Serialize\Serializer\Json $serializer = null
+        \Magento\Framework\Serialize\Serializer\Json $serializer = null,
+        FormatInterface $localeFormat = null
     ) {
         $this->convertor = $convertOrderFactory->create();
         $this->taxConfig = $taxConfig;
-        $this->serializer = $serializer ?: \Magento\Framework\App\ObjectManager::getInstance()->get(
+        $this->serializer = $serializer ?: ObjectManager::getInstance()->get(
             \Magento\Framework\Serialize\Serializer\Json::class
```

Comment:
You can also import that class. I recommend you to use alias `JsonSerializer` to be explicit what it does.

Reply:
The changes has been added, thank you for recommendation.